python/cudf/cudf/dataframe/column.py

```diff
@@ -48,14 +48,14 @@ def _concat(cls, objs, dtype=None):
         from cudf.dataframe.categorical import CategoricalColumn
 
         if len(objs) == 0:
-            if pd.api.types.is_categorical_dtype(dtype):
+            if dtype == np.dtype("object"):
```

Comment:
Should do `dtype = pandas.api.dtypes.pandas_dtype(dtype)` before here, and then check `dtype.type` instead. Should also catch if a user passes in `str` or `object`